File: src/test/scala/com/infotarget/codility/scala/PermCheckTest.scala
package com.infotarget.codility.scala

import org.scalatest.FunSuite

class PermCheckTest extends FunSuite {

  test("testSolution") {
    assert(1 === PermCheck.solution(Array(4, 1, 3, 2)))
    assert(0 === PermCheck.solution(Array(4, 1, 3)))
  }
}

File: src/test/java/com/infotarget/codility/java/PermCheckTest.java
package com.infotarget.codility.java;

import org.junit.Assert;
import org.junit.Test;

public class PermCheckTest {

    private PermCheck permCheck = new PermCheck();

    @Test
    public void testSolution() throws Exception {
        Assert.assertEquals(1, permCheck.solution(new int[]{4, 1, 3, 2}));
        Assert.assertEquals(0, permCheck.solution(new int[]{4, 1, 3}));
    }
}